assets/javascripts/discourse/initializers/extend-for-assigns.js

```diff
@@ -41,13 +41,37 @@ function extendTopicModel(api) {
   api.modifyClass("model:topic", {
     pluginId: PLUGIN_ID,
 
+    assignees() {
+      const result = [];
+
+      if (this.assigned_to_user) {
+        result.push(this.assigned_to_user);
+      }
+
+      const postAssignees = this.assignedPosts().map((p) => p.assigned_to);
+      result.push(...postAssignees);
+      return result;
```

Comment:
In this situation what I sometimes do is:


```javascript
assignees() {
  const assignees = this.assignedPosts().map((p) => p.assigned_to);
  if (this.assigned_to_user) {
    assignes.push(this.assigned_to_user);
  }
  return assignees;
}
```

It avoids various operations and makes the code smaller without sacrificing clarity IMO.